chemicalx/data/batchgenerator.py

```diff
@@ -149,18 +120,16 @@ def generate_batch(self, batch_frame: pd.DataFrame):
 
         labels = self._transform_labels(batch_frame["label"])
 
-        batch = DrugPairBatch(
-            batch_frame,
-            drug_features_left,
-            drug_molecules_left,
-            drug_features_right,
-            drug_molecules_right,
-            context_features,
-            labels,
+        return DrugPairBatch(
+            identifiers=batch_frame,
+            drug_features_left=drug_features_left,
+            drug_molecules_left=drug_molecules_left,
+            drug_features_right=drug_features_right,
+            drug_molecules_right=drug_molecules_right,
+            context_features=context_features,
+            labels=labels,
```

Comment:
Is there ever a time where `labels=False`? Perhaps this was a premature abstraction - I think we can actually remove this argument everywhere.

Reply:
I suppose some unsupervised learning could be a case.